### PR TITLE
Fix parsing of test runner's output messages

### PR DESCRIPTION
**Pull request by svor**
Signed-off-by: Valeriy Svydenko <vsvydenk@redhat.com>

### What does this PR do?
Checks for the end of test output messages.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/7338

### CODE REVIEW

**Comment by svor** [comment]
ci-test

**Review by tolusha — APPROVED** [review]

File: wsagent/che-core-api-testing/src/test/java/org/eclipse/che/api/testing/server/LineSplitterTest.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import org.eclipse.che.api.testing.server.framework.LineSplitter;
+import org.eclipse.che.api.testing.server.framework.LineSplitter.Consumer;
+import org.mockito.Mock;
+import org.mockito.testng.MockitoTestNGListener;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+
+@Listeners(MockitoTestNGListener.class)
+public class LineSplitterTest {
```
Comment (tolusha) [inline wsagent/che-core-api-testing/src/test/java/org/eclipse/che/api/testing/server/LineSplitterTest.java]: Add javadoc pls

Comment (svor) [reply wsagent/che-core-api-testing/src/test/java/org/eclipse/che/api/testing/server/LineSplitterTest.java]: added

**Comment by svor** [comment]
ci-build

**Review by evidolob — APPROVED** [review]

**Review by vparfonov — APPROVED** [review]

**Review by tolusha — APPROVED** [review]

**Review by musienko-maxim — APPROVED** [review]

**Pull request merged**